DOCUMENTATION.md

```diff
@@ -284,9 +291,73 @@ Revoke support access to an application.
 #### Examples:
     >> > resin.models.application.revoke_support_access('5685')
     'OK'
+## Build
+
+This class implements build model for Resin Python SDK.
+### Function: get(id)
+
+Get a specific build.
+
+#### Args:
+    id (str): build id.
+
+#### Returns:
+    dict: build info.
+
+#### Raises:
+    BuildNotFound: if build couldn't be found.
+### Function: get_all_by_application(app_id, include_logs)
+
+Get a specific build.
```

Comment:
~~This should be 'Get all builds for a specific application'~~ I see this is fixed in https://github.com/resin-io/resin-sdk-python/pull/60